admin/Model/Config/Source/Mode.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Pronko\LiqPayAdmin\Model\Config\Source;
```

Comment:
Better to use Pronko\LiqPayAdmin\Source namespace and Environment class name.

Reply:
What you mean under "Environment" class name.  

I should just change namespace to Pronko\LiqPayAdmin\Source and class have to has name Mode ? Did i understand you correct ? 

Sorry if my questions is stupid or primitive.

Reply:
It will be great to move this file to the Pronko\LiqPayAdmin\Source directory, and I thought (at least from my practice) to name a class "Environment" instead of a "Mode" because usually, this is about Sandbox Environment or Production Environment, however, Sandbox Mode also sounds reasonable.

Reply:
Ok, will be done.